research/deeplab/datasets/segmentation_dataset.py

```diff
@@ -85,10 +85,24 @@
     ignore_label=255,
 )
 
+# These number (i.e., 'train'/'test') seems to have to be hard coded
+# You are required to figure it out for your training/testing example.
+# Is there a way to automatically figure it out ?
+_ADE20K_INFORMATION = DatasetDescriptor(
+    splits_to_sizes = {
+        'train': 20210, # num of samples in images/training
+        'val': 2000, # num of samples in images/validation
+        'eval': 2,
```

Comment:
What is this split? Add comment for it? It seems that this split is not generated?

Reply:
Oh yes, this split is used for my own testing. I just forgot to remove it. I will turn it into a comment